.github/settings.yml

```diff
@@ -0,0 +1,119 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/
+repository:
+  topics: playframework, json, dsl, convenience-macros, json-ast, automatic-conversion, scala, scalajs, serialization, serializer
+  private: false
+  has_issues: true
+  has_projects: false
+  # We don't need wiki since the documentation lives in playframework.com
+  has_wiki: false
+  has_downloads: true
+  default_branch: master
+  allow_squash_merge: true
+  allow_merge_commit: false
+  allow_rebase_merge: false
+
+teams:
+  - name: core
+    permission: admin
+  - name: integrators
+    permission: write
+  - name: write-bots
+    permission: write
+
+branches:
+  - name: master
```

Comment:
These two rules are exactly the same, shouldn't we just use `*` for matching all branches?

Reply:
Yep, this is something to discuss. The existing difference is that master branch is more relaxed and it is not requiring the status checks and also it is not requiring the branch to be updated before merging.

I keep it separated here so that we can decide if we keep master like what we have today or if we make it more strict like the stable branches.

Reply:
And by the way, this is not consistent across our repositories. Some enforce the pull request status checks, some don't; some require the branch to be updated, some don't.